CODEOWNERS

```diff
@@ -58,7 +58,7 @@
 /specification/recoveryservicessiterecovery/ @avneeshrai
 /specification/redis/ @siddharthchatrolams @timlovellsmith
 /specification/relay/ @sethmanheim @v-ajnava
-/specification/resources/ @Tiano2017 @rajshah11 @vivsriaus
+/specification/resources/ @arm-template-deployments @rajshah11 @vivsriaus
```

Comment:
Does the team name need to be qualified with "Azure/"? Is there any way to test this out?

Reply:
Are you part of this group? Can you receive below notification?

Reply:
I was going to ask the same question. It looks like we need to qualify it. I'll update the PR.

---
In reply to: [509598826](https://github.com/Azure/azure-rest-api-specs/pull/11315#discussion_r509598826) [](ancestors = 509598826)

Reply:
@raych1 I checked with a team member who is not tagged in this PR and he received an notification.